offset:
  lf_q1_joint: 0
  lf_q2_joint: 0
  lf_q3_joint: 0
  lm_q1_joint: 0
  lm_q2_joint: 0
  lm_q3_joint: 0
  lr_q1_joint: 0
  lr_q2_joint: 0
  lr_q3_joint: 0
  rf_q1_joint: 0
  rf_q2_joint: 0
  rf_q3_joint: 0
  rm_q1_joint: 0
  rm_q2_joint: 0
  rm_q3_joint: 0
  rr_q1_joint: 0
  rr_q2_joint: 0
  rr_q3_joint: 0
init_pose_for_offset_tuner:
  lf_q1_joint: 0
  lf_q2_joint: 0
  lf_q3_joint: 0
  lm_q1_joint: 0
  lm_q2_joint: 0
  lm_q3_joint: 0
  lr_q1_joint: 0
  lr_q2_joint: 0
  lr_q3_joint: 0
  rf_q1_joint: 0
  rf_q2_joint: 0
  rf_q3_joint: 0
  rm_q1_joint: 0
  rm_q2_joint: 0
  rm_q3_joint: 0
  rr_q1_joint: 0
  rr_q2_joint: 0
  rr_q3_joint: 0
